packages/web/webpack.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
         new webpack.SourceMapDevToolPlugin({
             filename: '[file].map',
         }),
+        new webpack.ProvidePlugin({
+            process: 'process/browser',
```

Comment:
I don't see what this is being used for.

Reply:
We have some dependencies that use process.nextTick. This polyfills that.
(I don't remember which though)

Reply:
It is _stream_readable.js, which is the polyfill for the nodejs stream readable.